test/controllers/pages_controller_test.rb

```diff
@@ -116,4 +116,26 @@ class PagesControllerTest < ActionDispatch::IntegrationTest
     @page.reload
     refute @page.file.attached?
   end
+
+  test "should create a zip file" do
+    req_url = "#{@page.url}.zip"
+    expected_zip_file = "tmp/#{req_url}"
+
+    get "/#{req_url}"
+
+    assert File.file?(expected_zip_file)
+    File.delete expected_zip_file
```

Comment:
Why is necessary delete the file at the end of the test?

Reply:
@MatheusRich Just for a convenience.. If not, then the zip file will always be there on `tmp`. It's not something that needs to worry about for sure, but adding 1 line to make sure the environment is clear doesn't hurt either 😉

Reply:
Ok. I thought it was this.